### PR TITLE
Added the functionality to the Tabs

### PR DESCRIPTION
**Pull request by alejandrolue**

### CODE REVIEW

File: src/components/MainHeader/MainHeader.component.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import {
     ListItem,
     ListItemText,
     Toolbar,
-    Typography,
+    Typography, Box,
```
Comment (OG-Jons) [inline src/components/MainHeader/MainHeader.component.jsx]: reformat import

File: src/components/MainHeader/MainHeader.component.jsx [hunk 2]
```diff
@@ -189,6 +190,39 @@ const useStyles = makeStyles((theme) => ({
...
+            {value === index && (
+                <Box p={3}>
+                    <Typography>{children}</Typography>
+                </Box>
+            )}
+        </div>
+    )
+}
+
+TabPanel.propTypes = {
```
Comment (OG-Jons) [inline src/components/MainHeader/MainHeader.component.jsx]: propTypes not necessary

File: src/components/MainHeader/MainHeader.component.jsx [hunk 3]
```diff
@@ -31,6 +31,7 @@ import Tab from '@material-ui/core/Tab'
 import PeopleIcon from '@material-ui/icons/People'
 import EmojiPeopleIcon from '@material-ui/icons/EmojiPeople'
 import LanguageIcon from '@material-ui/icons/Language'
+import PropTypes from 'prop-types'
```
Comment (OG-Jons) [inline src/components/MainHeader/MainHeader.component.jsx]: remove import after removing PropTypes

**Review by OG-Jons — APPROVED** [review]
lgtm

**Review by Sl1van — APPROVED** [review]
lgtm

**Pull request merged**